### PR TITLE
Migrate database to Supabase

### PR DESCRIPTION
**Pull request by andrelandgraf**
## Summary (1-2 sentences)

PR to migrate database to Supabase

## Details (reason and description of the changes)

PlanetScale removed its free tier. This PR updates the Prisma schema for our migration to Supabase (Postgres).

### Implementation details (how was this change implemented?) (optional)

- Updates Prisma schema to run on Postgres
- Adds new `DIRECT_URL` environment variable (as suggested by Supabase, [see Prisma docs for more info](https://www.prisma.io/docs/orm/reference/prisma-schema-reference))

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -6,8 +6,9 @@ generator client {
 }
 
 datasource db {
-  provider     = "mysql"
+  provider     = "postgresql"
```
Comment (andrelandgraf) [inline prisma/schema.prisma:9]: Supabase uses Postgres

File: prisma/schema.prisma [hunk 2]
```diff
@@ -6,8 +6,9 @@ generator client {
...
   url          = env("DATABASE_URL")
+  directUrl    = env("DIRECT_URL")
```
Comment (andrelandgraf) [inline prisma/schema.prisma:11]: `DIRECT_UR`L used for Prisma migrations, which avoids connection pooling.

**Review by eiffelwong1 — APPROVED** [review]
LGTM

**Review by CamilingJS — APPROVED** [review]
LGTM

**Pull request merged**